File: tests/fixtures/e2e/python/good.py
from temporalio import workflow
from datetime import timedelta


@workflow.defn
class GreetingWorkflow:
    @workflow.run
    async def run(self, name: str) -> str:
        result = await workflow.execute_activity(
            greet,
            name,
            start_to_close_timeout=timedelta(seconds=10),
        )
        return result

File: tests/fixtures/e2e/rust/good.rs
use temporal_sdk::workflow;

#[workflow]
async fn greeting_workflow(name: String) -> Result<String, anyhow::Error> {
    let opts = ActivityOptions {
        start_to_close_timeout: Duration::from_secs(10),
        ..Default::default()
    };
    let result = execute_activity(greet_activity, name, opts).await?;
    Ok(result)
}
